x-pack/filebeat/module/rabbitmq/module.yml

```diff
@@ -0,0 +1,3 @@
+dashboards:
```

Comment:
This should be removed as no dashboards are in this PR. You can follow up with them in an other PR.